### DISCUSSION TITLE
How do I enabled MarshallOptions - RemoveUndefinedValues

### DISCUSSION THREAD

**Discussion by capistranc** [sensedeep/dynamodb-onetable#537] [Q&A]
OneTable makes it very hard for me to know whats going on. I cannot find out how to pass in a simple option. The documentation that does exists makes it look like I can pass in an object:
```
marshall: { removeUndefinedValues: true}
```

However this doesn't work and I have no way of knowing what contract onetable is expecting.

**Comment by mobsense** [comment]
Can you describe what you are trying to do and why you need to use the Marshall option?

**Reply by capistranc** [reply]
when i try to update DynamoDB I receive the error `Pass options.removeUndefinedValues=true to remove undefined values from map/array/set`

This should be a simple option to add but OneTable makes it impossible.

How does a person add any option to OneTable?

**Comment by dev-embedthis** [comment]
Can you describe more fully what you are doing?  OneTable removes undefined values by default.

If you construct your own document client, you can pass that to the OneTable constuctor.

```
const ddbDocClient = DynamoDBDocumentClient.from(client, {
  marshallOptions: {
    removeUndefinedValues: true, // Automatically remove undefined values
  },
});
```

But again, we don't use or need this as OneTable does this by default.
